### ISSUE TITLE
Float as a Default layout.

### ISSUE THREAD

**Issue by niksingh710** [material-shell/material-shell#310]
Description
When setting float as a default layout and going to an empty workspace shows a weird UI glitch.

**To Reproduce**
- Occurs on:
  - [x] X.Org
  - [ ] Wayland
- GNOME Shell version: 3.36
- Distro: Pop OS 20.04
- Steps to reproduce, if applicable:
1. Set Float as the default layout
2. Open any software then go to the bottom workspace 
3. You got the bug.
 
**Current commit**
Output from `git rev-parse --short HEAD`: `688ce24`

![Screenshot from 2020-09-21 20-45-10](https://user-images.githubusercontent.com/60490474/93796575-a6900780-fc58-11ea-9caf-3a2054d746d6.png)

**Comment by dwrolvink** [comment]
Can't reproduce for:

5.8.6-1-MANJARO
X.Org X11
Gnome Shell 3.36.6

**Comment by sebastian-xyz** [comment]
That is interesting 😄
I was able to reproduce it:

Fedora 32
Kernel 5.8.10
Wayland and Xorg

@dwrolvink Do you use transparency?

**Comment by dwrolvink** [comment]
@sebastian-xyz I use transparency in the panels, and terminator.

Perhaps I'm doing the steps wrong.

- Set float to default in settings
- Go to bottom workspace, make sure it's set to floating
- Go to top workspace, make sure it's set to floating
- Open new terminal
- Go to bottom workspace (by clicking the icon)

I tried with different applications, creating a new workspace instead of just clicking the bottom one, etc. No glitches.

Though I have to say, I did notice some buggy behavior on the bottom workspace, but I can't reproduce this specific one.
